src/s3/CMakeLists.txt

```diff
@@ -6,4 +6,6 @@ target_sources(
   getobject.cpp
   putobject.cpp
   deleteobject.cpp
-  headobject.cpp copyobject.cpp)
+  headobject.cpp
+  headbucket.cpp
+  copyobject.cpp)
```

Comment:
any need/use/convention to alphabetize these?

Reply:
I went ahead and sorted them alphabetically.  I also had GetObject_Test listed twice so I removed one of them.